x-pack/plugins/apm/server/routes/alerts/route.ts

```diff
@@ -34,6 +34,8 @@ const alertParamsRt = t.intersection([
   }),
   t.partial({
     groupBy: t.array(t.string),
+    useFilterQuery: t.string,
+    filterQuery: t.string,
```

Comment:
Why do we need `useFilterQuery` on the API? What about making `filterQuery` optional (and rename it to `kqlFilter`).

```suggestion
    kqlFilter?: t.string,
```

Reply:
`filterQuery` now renamed `kqlFilter` is already optional. I have removed `useFilterQuery` from the API as it is indeed not needed.